include/ttmlir/Dialect/TTIR/IR/TTIROps.td

```diff
@@ -172,8 +172,12 @@ def TTIR_DeallocOp : TTIR_Op<"dealloc"> {
 // TTIR top level named ops
 //===----------------------------------------------------------------------===//
 
+def 2Operands : ParamNativeOpTrait<"NOperands", "2">;
```

Comment:
Nit: just a suggestion, can we rename this numbers to letters? This O seems weirdly like zero, hence on a first look it looks like 20, 30, 40 operands...